lhotse/workflows/activity_detection/README.md

```diff
@@ -0,0 +1,156 @@
+# Activity Detection
+
+The Activity Detection module provides tools for detecting activity in audio recordings. It includes the following components:
+
+- `ActivityDetector` base class, designed to facilitate the integration of new activity detection methods in the future.
+- A runner class named `ActivityDetectionProcessor` for parallel execution of activity detection on the `RecordingSet``.
+- Two classes, `SileroVAD8k` and `SileroVAD16k`, which integrate the [Silero VAD](https://github.com/snakers4/silero-vad) model for activity detection.
+
+
+
+## Example usage in command line interface (CLI)
+
+1. Prepare the model for work:
+    ```bash
+    lhotse workflows activity-detection \
+    --model-name silero_vad_16k \
+    --chore
```

Comment:
Update this doc to be consistent with later changes in this PR

Reply:
The documentation has been updated to clarify the purpose of the `--force_download` flag